src/AutoRest.CSharp/Common/AutoRest/Communication/StandaloneGeneratorRunner.cs

```diff
@@ -83,9 +83,18 @@ await foreach (var file in workspace.GetGeneratedFilesAsync())
             }
         }
 
-        private static void WriteIfNotDefault(Utf8JsonWriter writer, string option, bool value)
+        private static void WriteIfEnumNotDefault<T>(Utf8JsonWriter writer, string option, T enumValue) where T : struct, Enum
```

Comment:
Since template is used, we don't need `Enum` in the name, e.g. original name is fine. And for bool value, we can use `WriteIfNotDefault(Utf8JsonWriter writer, string option, bool value)` as an overload.